src/main/java/com/gym/gymsystem/controller/TrainingTypeController.java

```diff
@@ -0,0 +1,26 @@
+package com.gym.gymsystem.controller;
+
+import com.gym.gymsystem.entity.TrainingType;
+import com.gym.gymsystem.service.TrainingTypeService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/training-types")
+public class TrainingTypeController {
+    private final TrainingTypeService trainingTypeService;
+
+    public TrainingTypeController(TrainingTypeService trainingTypeService) {
+        this.trainingTypeService = trainingTypeService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<TrainingType>> getAllTrainingTypes() {
```

Comment:
This endpoint is under authorization, was this done on purpose?

Reply:
you are right, but currently authorization is needed.
You have this configuration .anyRequest().authenticated()

Reply:
You're right, I missed it, I'll fix it, thanks